conjure-java/src/main/java/com/palantir/conjure/java/cli/ConjureJavaCli.java

```diff
@@ -189,6 +189,12 @@ public void run() {
                 description = "Generates compile-time safe builders to ensure all required attributes are set.")
         private boolean useStagedBuilders;
 
+        @CommandLine.Option(
+                names = "--excludeEmptyOptionals",
+                defaultValue = "false",
+                description = "Objects exclude empty optionals in serialization based on the conjure spec.")
+        private boolean excludeEmptyOptionals;
```

Comment:
This can just work with gradle-conjure? can you pass that like e.g. useImmutableBytes?

Reply:
That’s correct